cli/dcoscli/config/main.py

```diff
@@ -170,6 +174,17 @@ def _set(name, value):
     _check_config(toml_config_pre, toml_config)
 
     config.save(toml_config)
+
+    if not value_exists:
+        emitter.publish("[{}] set to '{}'".format(name, python_value))
+    elif old_value == python_value:
```

Comment:
Can we change python_value to new_value so it's obvious what's happening? I was a little confused ate first by "old_value" and "python_value".